app/states/testMulti.js

```diff
@@ -0,0 +1,82 @@
+import StackQuest from '../main'
+
+// Util functions for firebase database
+import {addCharacter, GamePlayers, onCharacterUpdate, updateCoordinates, updateOnCharacterMovement} from './firebase-database'
+const dummyCharId = `character${Math.floor(Math.random()*50)}`
+
+let map
+  , layer
+  , cursors
+  , fixedText
+
+const testState = {
+  preload() {
+    StackQuest.load.spritesheet('link', '../assets/link.png', 100, 50)
+    StackQuest.load.tilemap('testmap', 'maps/testmap.json', null, Phaser.Tilemap.TILED_JSON)
+    StackQuest.load.image('tiles', 'maps/terrain.png')
+  },
+
+  create() {
+    StackQuest.physics.startSystem(Phaser.Physics.P2JS)
```

Comment:
@kskkido Are we using the P2 Physics or Arcade Physics engine that Phaser provides? Past commits use the Arcade style, we should be consistent.